packages/smb/src/constants/paginators.ts

```diff
@@ -1,2 +1,3 @@
 export const OFFICES_PER_PAGE = 12
 export const NEGOTIATORS_PER_PAGE = 12
+export const NEGOTIATOR_OFFICE_PER_PAGE = 100
```

Comment:
Why need to show 100 per page ?

Reply:
We need to load all offices into Office SelectBox in Negotiator page for user to select office when creating a new negotiator.
This is just a temporary solution because SelectBox is not a good choice in this case, we need another component like AutoComplete or a SelectBox supporting searching feature. But @HollyJoyPhillips  confirm that we can leave as it is for now and we can address it later